draft-demarco-oauth-status-attestations.md

```diff
@@ -273,6 +273,24 @@ the public key contained in the Digital Credential;
 The technical and details about the `credential_pop` object
 are defined in the next section.
 
+## Status Attestation Request Errors
+
+In cases where a Status Attestation request is made for a Digital Credential that does not exist, has expired, been revoked, or is in any way invalid, or if the HTTP Request is compromised by missing or incorrect parameters, the Credential Issuer is required to respond with an HTTP Response. This response should have a status code of `400` and use `application/json` as the content type, including the following parameters:
+
+- `error`, REQUIRED. The value must be assigned one of the error types as specified in the OAuth 2.0 RFC [Section 5.2](https://tools.ietf.org/html/rfc6749#section-5.2);
```

Comment:
For the sake of clarity, I explicitly would provide a list of error codes that are useful in this specification. For example, it is not clear how to handle the case of an error response due to a Credential revocation. I don't see a clear error code in the RFC6749 that is clear for this scenario. We could put this in an issue and postpone this discussion if you want.

Reply:
in RFC6749 I see invalid_request, invalid_client, unauthorized_client ...